Source/JavaScriptCore/wasm/WasmTable.h

```diff
@@ -52,8 +51,18 @@ class Table : public ThreadSafeRefCounted<Table> {
     WTF_MAKE_NONCOPYABLE(Table);
     WTF_MAKE_TZONE_ALLOCATED(Table);
 public:
+    using MallocOps = VMMalloc;
```

Comment:
Lets add `DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(WasmTable)` and use `WasmTableMalloc` instead.